app/controllers/trainees/award_recommendations_controller.rb

```diff
@@ -6,9 +6,7 @@ def create
       if OutcomeDateForm.new(trainee).save! && trainee.submission_ready?
         trainee.recommend_for_award!
 
-        if FeatureService.enabled?(:integrate_with_dqt)
-          Dqt::RecommendForAwardJob.perform_later(trainee)
-        end
+        Dqt::RecommendForAwardJob.perform_later(trainee)
```

Comment:
So no guard here anymore? ```FeatureService.enabled?(:integrate_with_dqt)``` ?

Reply:
The job already has the guard clause built in - I should haven mentioned that.